test/conformance/kuttl/mutate/refactor/k10-minimum-retention/policy.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: k10-minimum-retention
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: k10-minimum-retention
+  annotations:
+    policies.kyverno.io/title: Minimum Backup Retention
+    policies.kyverno.io/category: Kasten K10 by Veeam
+    kyverno.io/kyverno-version: 1.6.2
+    policies.kyverno.io/minversion: 1.6.2
+    kyverno.io/kubernetes-version: "1.21-1.22"
+    policies.kyverno.io/subject: Policy
+    policies.kyverno.io/description: >-
+      K10 Policy resources can be validated to adhere to common compliance retention standards. 
+      Uncomment the regulation/compliance standards you want to enforce for according to GFS retention.
+      This policy deletes the retention value in the backup operation and replaces it with the specified retention.
+      Note: K10 Policy uses the GFS retention scheme and export operations default to use the retention of the backup operation.
+      To use different 
+      This policy can also be used go reduce retentions lengths to enforce cost optimization.
+spec:
+  schemaValidation: false
+  rules:
+  - name: k10-minimum-retention
+    match:
+      any:
+      - resources:
+          kinds:
+          - config.kio.kasten.io/v1alpha1/Policy
+    mutate: 
+      # Federal Information Security Management Act (FISMA): 3 Years 
+      #patchesJson6902: |-
+      #  - path: "/spec/retention"
+      #    op: replace
+      #    value: {"hourly":24,"daily":30,"weekly":4,"monthly":12,"yearly":3}
+      
+      # Health Insurance Portability and Accountability Act (HIPAA):  6 Years   
+      #patchesJson6902: |-
+      #  - path: "/spec/retention"
+      #    op: replace
+      #    value: {"hourly":24,"daily":30,"weekly":4,"monthly":12,"yearly":6}
+      
+      # National Energy Commission (NERC): 3 to 6 Years  
+      #patchesJson6902: |-
+      #  - path: "/spec/retention"
+      #    op: replace
+      #    value: {"hourly":24,"daily":30,"weekly":4,"monthly":12,"yearly":3}
+      
+      # Basel II Capital Accord: 3 to 7 Years 
+      #patchesJson6902: |-
+      #  - path: "/spec/retention"
+      #    op: replace
+      #    value: {"hourly":24,"daily":30,"weekly":4,"monthly":12,"yearly":3}
+      
+      # Sarbanes-Oxley Act of 2002 (SOX): 7 Years
+      #patchesJson6902: |-
+      #  - path: "/spec/retention"
+      #    op: replace
+      #    value: {"hourly":24,"daily":30,"weekly":4,"monthly":12,"yearly":7}
+      
+      # National Industrial Security Program Operating Manual (NISPOM): 6 to 12 Months 
+      #patchesJson6902: |-
+      #  - path: "/spec/retention"
+      #    op: replace
+      #    value: {"hourly":24,"daily":30,"weekly":4,"monthly":6}
```

Comment:
Do we want to test these?

Reply:
cc @chipzoller ?

Reply:
Yes, we do not want to modify the policies being tested in any of these cases since they come directly from kyverno/policies, and once we fix [this blocking issue](https://github.com/kyverno/website/issues/822) they will be incorporated back into that repo. We don't want any drift in between that time.